firestore/src/android/firestore_android.h

```diff
@@ -192,6 +215,10 @@ class FirestoreInternal {
   std::unordered_set<ListenerRegistrationInternal*> listener_registrations_;
 #endif  //  defined(_STLPORT_VERSION)
 
+  Mutex bundle_listener_mutex_;
+  std::vector<std::unique_ptr<LambdaEventListener<LoadBundleTaskProgress>>>
```

Comment:
It's probably a good idea to add an inline comment here to explain that the entire purpose of this vector is to have the `LambdaEventListener` objects destroyed when the `FirestoreInternal` is destroyed. Otherwise, it just looks odd to have this vector of unique pointers that are never read.

Reply:
Done.